modules/auxiliary/scanner/ssh/ssh_identify_pubkeys.rb

```diff
@@ -68,7 +70,11 @@ def initialize
   end
 
   def key_dir
-    datastore['KEY_DIR']
+    ::File.expand_path(datastore['KEY_DIR'])
```

Comment:
By "any of this," I meant the other cases of ```File.expand_path``` in the code. Didn't mean to sound snarky.